diff --git a/node_modules/cognito-at-edge/dist/index.js b/node_modules/cognito-at-edge/dist/index.js
index 8a3d2da..04a2343 100644
--- a/node_modules/cognito-at-edge/dist/index.js
+++ b/node_modules/cognito-at-edge/dist/index.js
@@ -95,6 +95,15 @@ class Authenticator {
             throw new Error('Expected params.logoutConfiguration.logoutUri to be a valid non-empty string starting with "/"');
         }
     }
+
+    /**
+     * This will explicitly Hydrate the JWKS store as per:
+     * https://github.com/awslabs/aws-jwt-verify#explicitly-hydrating-the-jwks-cache
+     */
+    async hydrateJwtCache() {
+        await this._jwtVerifier.hydrate();
+    }
+
     /**
      * Exchange authorization code for tokens.
      * @param  {String} redirectURI Redirection URI.
