diff --git a/node_modules/cognito-at-edge/dist/index.js b/node_modules/cognito-at-edge/dist/index.js
index 10d5d9b..0722c73 100644
--- a/node_modules/cognito-at-edge/dist/index.js
+++ b/node_modules/cognito-at-edge/dist/index.js
@@ -26,6 +26,7 @@ class Authenticator {
     _cookieSettingsOverrides;
     _logger;
     _jwtVerifier;
+    _redirectPath;
     constructor(params) {
         this._verifyParams(params);
         this._region = params.region;
@@ -53,6 +54,7 @@ class Authenticator {
         this._csrfProtection = params.csrfProtection;
         this._logoutConfiguration = params.logoutConfiguration;
         this._parseAuthPath = (params.parseAuthPath || '').replace(/^\//, '');
+        this._redirectPath = (params.redirectPath || '').replace(/^\//, '');
     }
     /**
      * Verify that constructor parameters are corrects.
@@ -510,7 +512,8 @@ class Authenticator {
         const { request } = event.Records[0].cf;
         const requestParams = (0, querystring_1.parse)(request.querystring);
         const cfDomain = request.headers.host[0].value;
-        const redirectURI = `https://${cfDomain}`;
+        const redirectURI = [`https://${cfDomain}`, this._redirectPath].filter((p) => p).join('/');
+        
         try {
             const tokens = this._getTokensFromCookie(request.headers.cookie);
             if (this._logoutConfiguration && request.uri.startsWith(this._logoutConfiguration.logoutUri)) {
